easysid.cpp

```diff
@@ -26,7 +41,15 @@ EXPORT void start_capture(const char *fn)
         delete wav;
         wav = NULL;
     }
-    wav = makePcmOutputWav(fn);
+    if (pcm == NULL) {
+        if (not initAudioOutput(DEFAULT_SAMPLING_RATE)) {
```

Comment:
I don't think this is going to work if you later call `set_sampling_parameters` with a `sample_rate` that does not match the default.